Demo/src/main/java/com/paypal/android/ui/paypalweb/PayPalCheckoutView.kt

```diff
@@ -58,23 +61,40 @@ fun PayPalWebView(
             .padding(horizontal = contentPadding)
             .verticalScroll(scrollState)
     ) {
-        Step1_CreateOrder(uiState, viewModel)
+        Step1_WebOrNativeCheckoutHeader(viewModel)
+        Step2_CreateOrder(uiState, viewModel)
         if (uiState.isCreateOrderSuccessful) {
-            Step2_StartPayPalWebCheckout(uiState, viewModel)
+            Step3_StartPayPalCheckout(uiState, viewModel)
         }
         if (uiState.isPayPalWebCheckoutSuccessful) {
-            Step3_CompleteOrder(uiState, viewModel)
+            Step4_CompleteOrder(uiState, viewModel)
         }
         Spacer(modifier = Modifier.size(contentPadding))
     }
 }
 
 @Composable
-private fun Step1_CreateOrder(uiState: PayPalWebUiState, viewModel: PayPalWebViewModel) {
+fun Step1_WebOrNativeCheckoutHeader(viewModel: PayPalCheckoutViewModel) {
+    val uiState: PayPalUiState by viewModel.uiState.collectAsStateWithLifecycle()
```

Comment:
I think this PR has some old changes in it. We just updated this UI code in the last PR that was merged.